neo-codap/src/graph/graph.tsx

```diff
@@ -344,24 +712,29 @@ export class GraphComponent extends React.Component<IGraphProps, IGraphState> {
             .text(yAttr ? yAttr.name : '')
             .on('click', () => this.setState({ yMenuIsOpen: true }));
 
-        svg.selectAll('circle')
-            .data(coordinates)
+        coordinates.forEach((coordArray, index) => {
+            svg.selectAll('circle' + index)
+            .data(coordArray)
             .enter().append('circle')
             .attr('cx', function (coords: { x: number, y: number }) {
                 return coords.x;
             })
             .attr('cy', function (coords: { x: number, y: number }) {
                 return coords.y;
             })
+            .attr('fill', colors[index % colors.length])
             .attr('r', kPointRadius)
             .attr('className', 'nc-point');
+        });
 
         // const {divWidth, divHeight} = this.props.size;
         return (
             <div className="neo-codap-graph">
                 {node.toReact()}
                 {this.renderXAxisPopover()}
                 {this.renderYAxisPopover()}
+                {this.state.graphMenuIsOpen ? this.renderGraphPopover() : null}
```

Comment:
Is the test necessary?  I would have thought that whether or not the menu is open is handled properly by the popover, as it apparently is for the X and Y axes.

Reply:
It works a little differently than the x/y axis as the popover doesn't handle the initial click because we want the svg element to get the mouse events so that the hover and click on the data point circles are still enabled.  I'll document this.